planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/src/mpt_optimizer.cpp

```diff
@@ -584,8 +583,8 @@ std::vector<Bounds> MPTOptimizer::getReferenceBounds(
     if (
       lat_bounds_0[0] == lat_bounds_0[1] || lat_bounds_1[0] == lat_bounds_1[1] ||
       lat_bounds_2[0] == lat_bounds_2[1]) {
-      ROS_WARN_THROTTLE(1.0, "[Avoidance] Could not find driveable area for %i th point", cnt);
-      ROS_INFO_COND(is_showing_debug_info_, "Path is blocked at %i ", cnt);
+      RCLCPP_WARN(rclcpp::get_logger("MPTOptimizer"), "[Avoidance] Could not find driveable area for %i th point", cnt);
```

Comment:
Sorry, I missed your commend. Forget about this.

Reply:
This may be solved by this?

```
auto clock = rclcpp::Clock(RCL_ROS_TIME);
RCLCPP_WARN_THROTTLE(
    rclcpp::get_logger("MPTOptimizer"), 
    clock, 
    std::chrono::milliseconds(1000).count(), 
    "[Avoidance] Could not find driveable area for %i th point", cnt);
```

Reply:
done in https://github.com/tier4/Pilot.Auto/pull/83/commits/dbd1c440b7cfc65e7994c5216708e88c27aa4916